frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -954,7 +954,8 @@ function ReaderPaging:_gotoPage(number, orig_mode)
     end
     if number > self.number_of_pages or number < 1 then
         logger.warn("wrong page number: "..number.."!")
-        return false
+        number = self.number_of_pages
```

Comment:
Font size changes cause invalid page numbers. This function only protects against invalid links or something like that, not against invalid `last_page` settings. I think that's probably a more general issue, because what is you cut off a few pages from a PDF or something?